### ISSUE TITLE
Change the markdown renderer.

### ISSUE THREAD

**Issue by HaoZeke** [gatsbyjs/gatsby#3092]
Is it possible to get pandoc markdown compilation through [simple-pandoc](https://www.npmjs.com/package/simple-pandoc)?

**Comment by KyleAMathews** [comment]
Absolutely. You'd need to build a transformer plugin like gatsby-transformer-remark. They're fairly straightforward to build. Happy to give tips if needed and there's some docs on the process plus plenty of similar plugins to look at.
